msteams-platform/concepts/messaging-extensions/create-extensions.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Action based messaging extensions
+description: Describes how to develop action based messaging extensions
+keywords: teams messaging extensions messaging extensions search
+ms.date: 11/07/18
+---
+
+# Initiate actions with messaging extensions (Developer preview)
```

Comment:
The convention for features in developer preview is to have a "> [!NOTE]" near the top of the text, not in parens.

Reply:
Done.